src/kinova_gen3_mujoco_config/description/picknik_kinova_gen3.xacro

```diff
@@ -680,6 +680,7 @@
         <param name="mujoco_model">description/mujoco/scene.xml</param>
         <param name="mujoco_model_package">kinova_gen3_mujoco_config</param>
         <param name="render_publish_rate">10</param>
+        <param name="lidar_publish_rate">10</param>
```

Comment:
```suggestion
```
Remove this line and we'll wait for Bilal's change to make lidar_publish_rate optional before we merge this